apps/desktop/src/components/AddressPill/AddressPill.tsx

```diff
@@ -109,7 +110,7 @@ export const AddressPill = memo(
               </PopoverBody>
             </PopoverContent>
           </Popover>
-          {showIcons && (
+          {showIcons && address.type !== "smart_rollup" && (
```

Comment:
maybe we could temporarily use the same icon as for the contract?

Reply:
I think it can be confusing for the user